web/bower.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "material-dashboard-react",
```

Comment:
(Minor) For all metadata files, should appropriately update the metadata e.g. name, homepage, authors, description